### PR TITLE
LPS-203383 Add permission action in bulk for static pages without custom success message

### PR DESCRIPTION
**Pull request by Mikellorza**
subtask: https://liferay.atlassian.net/browse/LPS-203383
story: https://liferay.atlassian.net/browse/LPS-200677

### CODE REVIEW

File: portal-impl/src/portal.properties [hunk 1]
```diff
@@ -7211,7 +7216,7 @@
     #
     # Env: LIFERAY_FEATURE_PERIOD_FLAG_PERIOD__UPPERCASEL__UPPERCASEP__UPPERCASES__MINUS__NUMBER1__NUMBER9__NUMBER4__NUMBER3__NUMBER6__NUMBER2__PERIOD_TYPE
     #
-    feature.flag.LPS-194362.type=release
+    feature.flag.LPS-194362.type=beta
```
Comment (ealonso) [inline portal-impl/src/portal.properties:7219]: @Mikellorza I think this change is not related with this pr

Comment (Mikellorza) [reply portal-impl/src/portal.properties]: Fixed, thanks letting me know

**Review by lfbesada — APPROVED** [review]
LGTM

**Comment by sandrodw3** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
ci:forward:force
